File: CSharp/Unit1/01.Primitive-Data-Types-and-Variables/13.NullValue/NullValue.cs
/*
 *Create a program that assigns null values to an integer and to double variables. 
 *Try to print them on the console, try to add some values or the null literal to them and see the result.
 */
using System;
class NullValue
{
    static void Main()
    {
        int? a = null;
        double? b = null;
        Console.WriteLine(a);
        Console.WriteLine(b);
        a = a + 5;
        b = b + 5.3;
        Console.WriteLine(a);
        Console.WriteLine(b);
    }
}

File: Java/IntroJava/01.Primitive-Data-Types-and-Variables/src/NullValue.java
/*
 *Create a program that assigns null values to an integer and to double variables. 
 *Try to print them on the console, try to add some values or the null literal to them and see the result.
 */
public class NullValue {

	public static void main(String[] args) {
		int a = 0;//null;
        double b = 0;//null;
        System.out.println(a);
        System.out.println(b);
        a = a + 5;
        b = b + 5.3;
        System.out.println(a);
        System.out.println(b);
	}

}
